patches/@graphql-tools+code-file-loader+6.3.2-alpha-f1d7b3c2.0.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/node_modules/@graphql-tools/code-file-loader/index.js b/node_modules/@graphql-tools/code-file-loader/index.js
+index 7f34ec1..9660b6d 100644
+--- a/node_modules/@graphql-tools/code-file-loader/index.js
++++ b/node_modules/@graphql-tools/code-file-loader/index.js
+@@ -244,8 +244,11 @@ class CodeFileLoader {
+                 const content = await readFile(normalizedFilePath, { encoding: 'utf-8' });
+                 const sources = await graphqlTagPluck.gqlPluckFromCodeString(normalizedFilePath, content, options.pluckConfig);
+                 if (sources.length) {
++                    
++                    const document = graphql.concatAST(sources.map(source => graphql.parse(source.body, options)))
+                     return {
+-                        document: graphql.concatAST(sources.map(source => graphql.parse(source, options))),
++                        document,
++                        rawSDL: sources.map(source => source.body).join(`\n#-#\n`),
```

Comment:
The rawSDL is used for building the operation string to Type mapping. Since multiple `gql` calls are merged into a single document `\n#-#\n` is used for separating them later.

If we don't provide `rawSDL` at all, something very weird is happening and we end up with something weird similar to this: https://github.com/ardatan/graphql-tools/pull/3124 

For three `gql` calls within a single document:
```
[Object object]\n\n[Object object]\n\n[Object object]
```